### ISSUE TITLE
System Tray icon does not show when starting in Win7 using scheduled task workaround

### ISSUE THREAD

**Issue by GoogleCodeExporter** [soul4soul/peerblock#194]
```
What steps will reproduce the problem?
1.Use the start with windows workaround for Win7


What is the expected output? What do you see instead?
I would expect to see the tray icon and it normally isn't available. I 
know Peer block is running because if I run it from the start menu it show 
IP addresses in the log from before the application was (re)launched. 
Windows also seems to think it's not loaded as you get the (really 
annoying!) UAC message when trying to start it.

What version of PeerBlock are you using? On what operating system? 32- or
64-bit? Win7 Pro 64bit. PeerBlock 1.0(r202) (installed via the installer, 
not the zip file)


Please provide any additional information below.  Make sure to attach
peerblock.log and/or any screenshots that would help explain your problem.


```

Original issue reported on code.google.com by `mpor...@gmail.com` on 7 Nov 2009 at 11:23

Attachments:
- [PeerBlock Error.jpg](https://storage.googleapis.com/google-code-attachments/peerblock/issue-194/comment-0/PeerBlock Error.jpg)

**Comment by GoogleCodeExporter** [comment]
```
Hi,

I'm also using Peerblock 1.0(r202) and if I start the prog via Task scheduler I 
can
see the tray icon.
Can you check the task scheduler properties (see screen shots attached)?
```

Original comment by `cybermcm@gmail.com` on 8 Nov 2009 at 6:27

Attachments:
- [task_scheduler_1.png](https://storage.googleapis.com/google-code-attachments/peerblock/issue-194/comment-1/task_scheduler_1.png)
- [task_scheduler_2.png](https://storage.googleapis.com/google-code-attachments/peerblock/issue-194/comment-1/task_scheduler_2.png)

**Comment by GoogleCodeExporter** [comment]
```
I've noticed this same behavior, though it wasn't doing this when I first 
installed
it on Win7 Pro x64.  I gave PeerBlock a 1 minute delay before starting, though, 
and
it looks like that may have served as a workaround.  It appears that PeerBlock 
is
starting so early that the system tray icon doesn't instantiate.  Other 
programs that
start up using the same method (e.g. Secunia PSI) don't appear to have this 
problem.
```

Original comment by `KungFu...@gmail.com` on 9 Nov 2009 at 8:06

**Comment by GoogleCodeExporter** [comment]
```
I marked this in issue 326 as the icon disappearance occurs for other issues as 
well, such as updating (in one case). 
```

Original comment by `LDru...@gmail.com` on 14 Jun 2010 at 12:51

**Comment by GoogleCodeExporter** [comment]
```
Issue 292 has been merged into this issue.
```

Original comment by `nightstalkerz` on 26 Aug 2010 at 9:52
